### DISCUSSION TITLE
DagBag does not see part of the dags

### DISCUSSION THREAD

**Discussion by snowthejon** [apache/airflow#30354] [General]
So, I'm trying to test my dags. I'm retrieving my dags from DagBag and specifying the folder where dags are situated.

```
dagbag = DagBag(dag_folder="./dags/concreteDirWithDags")
```

The way for dag_folder is valid, file names has valid .py extension, code is valid without any syntax errors inside. 
There are no exceptions while starting tests for my dags, but dagbag does not see parth of the dags in the dag_folder directory.

**Comment by kobethuwis** [comment]
I don't think relative paths work there. Where is this code running? How is your dag_folder synced to the Airflow instance? You should use an absolute path which you can obtain by navigating to the folder in question and running `pwd`
